app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.ithaca_transit_android_v2.states
+
+import com.example.ithaca_transit_android_v2.models.Coordinate
+import com.example.ithaca_transit_android_v2.models.Location
+
+// Parent Class
+sealed class SearchState(
+    val startLocation: Location = Location("", Coordinate(0.0, 0.0), ""),
+    val endLocation: Location = Location("", Coordinate(0.0, 0.0), "")
+) {}
+
+// Default UI when launching app - only a search bar (no interactions so far)
```

Comment:
what does "no interactions so far" mean?

Reply:
No interaction means that the user has yet to click on the search bar. I have removed it because it is self-explanatory